resources/public/api/conf/3.0/submit_foreign_property_accounting_adjustments.raml

```diff
@@ -3,53 +3,53 @@ is:
   - headers.contentTypeJson
   - headers.testScenarioHeader
   - errors.formatNino
-  - errors.formatBsasId
-  - errors.formatAdjustmentValue
-  - errors.formatCountryCode
-  - errors.ruleCountryCode
-  - errors.ruleRangeInvalid
-  - errors.incorrectOrEmptyBodySubmitted
+  - errors.formatCalculationId
+  - errors.formatValue
   - errors.ruleBothExpensesSupplied
   - errors.ruleTypeOfBusinessIncorrect
```

Comment:
in the description for `ruleTypeOfBusinessIncorrect` states `self-employment` employment in the description. It needs to say `foreign`